ethereum/package.json

```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "@iarna/toml": "^2.2.5",
     "@openzeppelin/contracts": "^4.1.0",
-    "@openzeppelin/test-helpers": "^0.5.10",
+    "@openzeppelin/test-helpers": "https://github.com/Snowfork/openzeppelin-test-helpers.git",
```

Comment:
Depending on main/master may break tests in the future – can we lock this down to a specific commit at least?

Reply:
https://github.com/Snowfork/snowbridge/pull/492